src/CommunityToolkit.Maui.MediaElement/PlatformView/MediaManager.android.cs

```diff
@@ -210,7 +210,8 @@ public void OnPlayerStateChanged(bool playWhenReady, int playbackState)
 
 		if (playbackState == IPlayer.StateReady)
 		{
-			mediaElement.Duration = TimeSpan.FromMilliseconds(player.Duration);
+			mediaElement.Duration = TimeSpan.FromMilliseconds(
+				player.Duration == C.TimeUnset ? 0 : player.Duration);		
```

Comment:
can we extract it to be an extension method? I can see value in doing that, since this is something very particular to exoplayer

Reply:
Also, why not make it a little bit more fool proof and not only check for that specific value? I would assume anything less than 0 would be invalid as a duration